db/migrate/20160510173147_create_peer_reviews.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePeerReviews < ActiveRecord::Migration
+  def change
+    create_table :peer_reviews do |t|
+      t.integer :reviewer_id, null: false
```

Comment:
I think you want `t.references` here. http://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/TableDefinition.html#method-i-references